src/TingBundle/Cache/MetadataWarmer.php

```diff
@@ -95,7 +95,8 @@ public function warmUp($cacheDir)
             $cacheDir,
             $this->cacheFile
         );
-        $metadataCacheGenerator->createCache($repositories);
+
+        return [$metadataCacheGenerator->createCache($repositories)];
```

Comment:
Not sure that it's judicious to preload the `ting.php` cache file?
@vgrebccm or @xavierleune could you confirm? If not an empty array can be returned.

Reply:
I don't see any issue preventing us to preload theses cache files. Do you have something in mind ?